_layouts/progress-page.html

```diff
@@ -328,26 +328,30 @@ <h3 class="milestone-quarter-heading" id="milestone-q3">Quarter 3, 2020 (July &n
 		</div>
 	</div>
 	
-	<div id="action-1-3" class="usa-grid milestone-progress-row milestone-current">
+	<div id="action-1-3" class="usa-grid milestone-progress-row milestone-current status-partial-complete">
 		<ul class="usa-width-one-third milestone-meta">
 			<li><a href="{{ site.baseurl }}/action-plan/#action-1-identify-data-needs-to-answer-priority-agency-questions">Milestone 1.3</a>. <strong>By September 30, 2020</strong>, develop a plan to engage the stakeholders needed to access or acquire the data needed to answer priority questions.</li>
 		</ul>
 		<div class="usa-width-two-thirds milestone-status">
 			<div class="milestone-status-classification">
 				This milestone has been <span class="status-text"><strong>partially COMPLETED</strong></span>. As of September 30, 2020:
 			</div>
-			<div class="milestone-status-metric">
-				<div class="milestone-status-metric-number">
-					2 ➧
-				</div>
-				<div class="milestone-status-metric-details">
-					CFO-Act agencies identified data needs for answering priority questions in the interim learning agenda. Two CFO-Act agencies are in progress. OMB review of this milestone is currently ongoing.   
-				</div>
-			</div>
 		</div>
 	</div>
 	
-	<div id="action-2-3" class="usa-grid milestone-progress-row milestone-current">
+	  
+	  <div id="action-17-1" class="usa-grid milestone-progress-row milestone-current status-partial-complete">
```

Comment:
You have 2 definitions of `action-17-1` as the ID, which should not occur for HTML. ID's should be unique. You could change this to be something like "action-17-1-oct" or some other representation, but I'm not sure if that would have an effect on your styling...